### PR TITLE
set `lower_is_better` property in `InstantiationBase.make_objectives`

### PR DESCRIPTION
**Pull request by lyhyl**

### CODE REVIEW

**Comment by EugenHotaj** [comment]
@lyhyl Thanks for the PR! Could you update the title to be more descriptive and fix the comment I left in the code?

**Comment by lyhyl** [comment]
> @lyhyl Thanks for the PR! Could you update the title to be more descriptive and fix the comment I left in the code?

OK, I have updated the title. But I do not know what comment I should fix, I did not found any comment in the code. Did I miss something?

**Comment by EugenHotaj** [comment]
@lyhyl do you not see my code review comment about refactoring the code slightly when you navigate to the "Files Changed" tab? (It also shows up right below my previous comment).

**Comment by lyhyl** [comment]
> @lyhyl do you not see my code review comment about refactoring the code slightly when you navigate to the "Files Changed" tab? (It should also show up right below my previous comment).

hmmm... I didn't see any comments.
image
image

File: ax/service/utils/instantiation.py [hunk 1]
```diff
@@ -438,6 +438,7 @@ def make_objectives(cls, objectives: Dict[str, str]) -> List[Objective]:
                     metric=cls._make_metric(
```
[the remark this answers is not in the document]
Comment (EugenHotaj) [reply ax/service/utils/instantiation.py:438]: @lyhyl sorry about that, apparently I had forgotten to click "submit" (have not used the code review in GH before). Let me know if you see it now.

[the remark this answers is not in the document]
Comment (lyhyl) [reply ax/service/utils/instantiation.py]: Okay, I can see it now. I have updated the code following the naming style of your team.

**Comment by lena-kashtelyan** [comment]
Doing the review and merging of the PR internally; @lyhyl nothing to do for you –– I'll take care of the rest here!

**Pull request closed without merging**